src/components/ftb-stage-table/ftb-stage-table.component.tsx

```diff
@@ -226,27 +227,50 @@ export class FtbStageTable {
   }
 
   private renderBody() {
-    let sliceStart = 0;
-    let sliceEnd = this.stage.table.length;
+    let res = [];
```

Comment:
1. `const`, not `let` 
2. rename variable to make code more readable. something like `displayingRows`